buildenv/jenkins/common/pipeline-functions

```diff
@@ -132,23 +132,29 @@ def build_with_slack(JOB_NAME, PARAMETERS) {
     def MESSAGE = ''
     def JOB = build job: JOB_NAME, parameters: PARAMETERS, propagate: false
 
-    if (JOB.resultIsWorseOrEqualTo('UNSTABLE')) {
```

Comment:
Is there a reason why we removed this `if`? We don't need to get the DOWNSTREAM_JOB_NUMBER or URL is the build passes, we only use it in the slack message if we are FAILED or UNSTABLE